glslc/README.asciidoc

```diff
@@ -249,9 +251,29 @@ the current working directory.
 Requests that the compiler place source-level debug information into the object
 code, such as identifier names and line numbers.
 
+This option restrains `-O` from turning on the strip-debug-info optimization
+pass.
+
 NOTE: Currently this option has no effect.  Full functionality depends on
 glslang support for generating debug info.
 
+==== `-O0`, `-Os`
+
+`-O` specifies which optimization level to use:
+
+* `-O0` means "no optimization". This level compiles the fastest and generates
+  the most debuggable code.
+* `-Os` enables optimizations to reduce code size. This level runs the
+  following optimization passes one by one:
+  - strip-debug-info: removes all debug instructions. (This pass will be turned
```

Comment:
Removed.

But I think we still need a way to tell the user what exactly we are running in case they are interested?